files/jquery.drawsvg/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "jquery.drawsvg",
+  "repo": "lcdsantos/jquery-drawsvg",
+  "files": {
+    "include": ["jquery.drawsvg.js", "jquery.drawsvg.min.js"]
```

Comment:
These files dont exist in the repo https://github.com/lcdsantos/jquery-drawsvg/tree/1.0.0
The bot will fail